test/querySelector.spec.js

```diff
@@ -1041,3 +1042,39 @@ describe('basic test suite', function () {
     ])
   })
 })
+
+describe('ancestor selector', () => {
+  const defaultDiv = {
+    tagName: 'SELECT',
+    classList: ['selectBox1']
+  }
```

Comment:
This variable should be called `defaultSelect` since it's a select, not a div.

Reply:
@nolanlawson Updated the variable name. Please merge the PR if you are good with the changes. Thanks